Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -569,14 +621,15 @@ private void saveControlPosition(int sharedPrefsId, int x, int y)
 	 * InputOverlayDrawableButton. Simply add it to the HashSet of overlay items and wait
 	 * for Android to call the onDraw method.
 	 * 
-	 * @param context  The current {@link Context}.
-	 * @param resId    The resource ID of the {@link Drawable} to get the {@link Bitmap} of.
-	 * @param buttonId Identifier for determining what type of button the initialized InputOverlayDrawableButton represents.
+	 * @param context        The current {@link Context}.
+	 * @param defaultResId   The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Default State).
+	 * @param pressedResId   The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Pressed State).
```

Comment:
because in other places the format was to align the description vertically, and they used different size of spaces. I am just saying because this is confusing for anyone wants to change the java code. Maybe better change everything to one style later.

for now I will change all of them back to single space